sdk/python/kfp/v2/components.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 The Kubeflow Authors
```

Comment:
nit: change these to 2022 :)

Reply:
Just did this in a separate PR: https://github.com/kubeflow/pipelines/pull/7401